server/bench-wrk/queries.graphql

```diff
@@ -0,0 +1,128 @@
+
+# A query which carries a lot of data
```

Comment:
Thanks for documenting the queries. It would be even better going forward when we add a query to answer the question: what do we expect to learn from measuring this? i.e. what's our hypothesis, for instance "a large payload returned from postgres to exercise our parsers and the GC (parameters such as nursery size might affect this)". This sort of documentation can be added to later as we learn more about the system as well.